WalkingChallenge/LoginViewController.swift

```diff
@@ -65,7 +65,13 @@ extension LoginViewController: FBSDKLoginButtonDelegate {
       alert(message: "Error logging in: \(error)", style: .cancel)
       return
     }
-    AppController.shared.login()
+    
+    if(FBSDKAccessToken.current() == nil){
```

Comment:
I would put this check into the above guard so the user gets an alert when login failed and so the main logic isn't wrapped in an `else`